lib/operations/command_v2.js

```diff
@@ -20,6 +20,7 @@ class CommandOperationV2 extends OperationBase {
     this.readConcern = resolveReadConcern(parent, this.options);
     this.writeConcern = resolveWriteConcern(parent, this.options);
     this.explain = false;
+    this.decorateWithCollation = true;
```

Comment:
`createIndexes` puts the collation into each index in the `indexes` property when building the command; it shouldn't be at the root level, so we need a way to turn this off now that `createIndexes` is using `CommandOperationV2`.